miniwallet/contracts/mocks/TestERC1155.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+/// @custom:security-contact dev@modulo.so
+contract TestERC1155 is ERC1155, AccessControl {
+    bytes32 public constant URI_SETTER_ROLE = keccak256("URI_SETTER_ROLE");
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+
+    constructor(uint256[] memory tokenIds, uint256[] memory amounts)
+        ERC1155("ipfs://testerc1155.modulo.so/")
```

Comment:
that's invalid ipfs address

Reply:
Understood.
This does not impact testing and is used as a placeholder only.
If we have a  pa preferred ipfs adress and I can update.

Reply:
@polymorpher Can you provide a better (valid) ipfs url which you’d like me to use instead of `ipfs://testerc1155.modulo.so/` 

for example for the valid URI’s used for the mini721 and mini1155 are
```
MINI721_DEPLOY_BASE_URI=ipfs://Qmc8DVEthq7cZMTMyZ2NQ8dHkG99n549DMBwNzAypQgXe1/Mini721/
```
and

```
MINI1155_DEPLOY_BASE_URI=ipfs://Qmc8DVEthq7cZMTMyZ2NQ8dHkG99n549DMBwNzAypQgXe1/Mini1155/
```
would you like me to hardcode these values?

Reply:
if the test data is not populated at an ipfs location already, empty seems more reasonable